server/controllers/intreview/cancalIntreview.ts

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable space-before-blocks */
+import { Response } from 'express';
+import { RequestType, mailSender } from '../../utils';
+import Interviewee from '../../database/Models/Interviewee';
+import Interviewer from '../../database/Models/Interviewer';
+import User from '../../database/Models/User';
+
+const cancalInterview = async (req: RequestType, res: Response) => {
+  // use vaildation for params after marge
+  const { interviewId } : any = req.params;
+  const { userInfo } = req;
+
+  const user = await User.findOne({ _id: userInfo?.id }, { email: 1, _id: 1, name: 1 });
+  const { email } = user;
+
+  // eslint-disable-next-line max-len
+  let collectionName: any; // the name of the collection (intrviewee or interviewer)
+
+  // // query about role by _id
+  if (userInfo){
```

Comment:
It will not reach this controller if there is no userInfo so no need for this condition